README.md

```diff
@@ -0,0 +1,39 @@
+QuOIL
+=====
+
+QuOIL checks the quality of the internet link by logging the ping statistics to a specified server at defined intervals.  The results can be saved to a file, that can then be analysed by other programs.
+
+The usage is as follows:
+
+`quoil [-h | --help]`
+
+* This prints a summary of the usage.
+
+`quoil [--interval sec] [--number nr] [--repeat nr [--wait min]] <ip_to_ping> [log_file_name]`
+
+* *\-\-interval sec* sets the number of seconds between each ping.  If it is not specified, it defaults to *:interval_between_pings_sec* specified in [**config/config.exs**](config/config.exs) (*\-\-interval* can be shortened to *-i*).
+
+* *\-\-number nr* sets the number of pings in each set used to generate the ping statistics.  If it is not specified, it defaults to *:number_of_pings* specified in [**config/config.exs**](config/config.exs) (*\-\-number* can be shortened to *-n*).
+
+* *\-\-repeat nr* sets the number of times that the ping sets are to be repeated.  If it is not specified, it defaults to *nil* (*\-\-repeat* can be shortened to *-r*)
```

Comment:
what does `nil` mean in this context? I'm assuming `0`? Better to be more explicit in the user facing documentation about what the effect of a config option is